src/contents/shared/ProfileEdit.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { Component } from 'react';
+import fire from '../../config/Fire';
+import { ProfileEditView } from './ProfileEditView';
+import firebase from 'firebase';
+import { storage } from '../../config/Fire';
```

Comment:
You don't need to change the config file, as you can simply call: `fire.storage()` like we did with the database. So please edit the config file, as it was before.

Reply:
Done!